addons/web/static/src/search/cog_menu/cog_menu.xml

```diff
@@ -10,7 +10,9 @@
                     </button>
 
                     <t t-set-slot="content">
+
                         <t t-if="state.printItems.length">
+                            <div role="separator" class="dropdown-divider"/>
```

Comment:
what if there's nothing before?

Reply:
We have a CSS rule to hide it if it's the first child.

Reply:
Not the kind of thing we like, but if it makes the js/xml logic simpler, why not